jcommune-view/jcommune-web-view/src/test/java/org/jtalks/jcommune/test/AdministrationControllerTest.groovy

```diff
@@ -213,4 +221,37 @@ class AdministrationControllerTest extends Specification {
             groups.assertGroupUserPage(page2, groupUserList.subList(20, 40))
             groups.assertGroupUserPage(page3, groupUserList.subList(40, 45))
     }
+
+    def 'in searchlist should be users not in group with id and with username or email matching pattern'() {
```

Comment:
This can be a much simpler test - it should just check that everything works together. The rest has to be tested on Dao level. 

Also it should check the validation if it exists on the controller level.